docs/examples/composer_app/composer.json

```diff
@@ -0,0 +1,6 @@
+{
+    "require": {
+        "monolog/monolog": "1.3",
+        "marcelog/pami": "dev-master"
```

Comment:
`^1.80`, just to be sure. Won't work until 1.80 is released, but then it will always work.

Also, by default composer will not install `dev-master`, unless it is specified as `dev-master@dev`, because of its stability rules.